src/sinol_make/contest_types/__init__.py

```diff
@@ -0,0 +1,19 @@
+import os
+import yaml
+
+from sinol_make.contest_types.default import DefaultContest
+from sinol_make.contest_types.oi import OIContest
+from sinol_make.interfaces.Errors import UnknownContestType
+
+
+def get_contest_type():
+    with open(os.path.join(os.getcwd(), "config.yml"), "r") as config_file:
+        config = yaml.load(config_file, Loader=yaml.FullLoader)
+        contest_type = config.get("sinol_contest_type", "default")
+
+    if contest_type == "default":
+        return DefaultContest()
+    elif contest_type == "OI":
+        return OIContest()
+    else:
+        raise UnknownContestType(f'Unknown contest type "{contest_type}"')
```

Comment:
We should create an example `config.yml` that also serves as a documentation. Basically a sample `config.yml` with a lot of comments that explains possible options and logic/behavior. What do you think?

Reply:
Yes, that’s a good idea

Reply:
I added an example config